theano/configdefaults.py

```diff
@@ -430,7 +430,7 @@ def f(suffix=suffix):
         except OSError:
             rc = 1
         if rc != 0:
-            _logger.warning("conda g++ not available, use `conda install m2w64-toolchain`")
+            _logger.warning("g++ not available, if using conda: `conda install m2w64-toolchain`")
```

Comment:
I'm not sure about this warning. I think it will be triggered too often. People not using conda could need the code after this and won't have mkl. I see 2 possibles fix:

1) Do the check for conda and keep the warning here. I prefer this option, I think it is fine to have only one place with a check for conda. This will ensure that conda user use mkl.
2) Just record here that mkl isn't available. Then if we can't find a blas in this function, raise this warning. So mostly, postpone this warning to the end of this method in case we don't find a blas.